package.json

```diff
@@ -159,9 +159,9 @@
     "@babel/plugin-transform-runtime": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.18.6",
-    "@folio/eslint-config-stripes": "^5.0.0",
-    "@folio/stripes": "^8.0.0",
-    "@folio/stripes-cli": "^2.0.0",
+    "@folio/eslint-config-stripes": "^7.0.0",
+    "@folio/stripes": "^9.0.0",
+    "@folio/stripes-cli": "^3.0.0",
     "@folio/stripes-data-transfer-components": "^5.0.0",
```

Comment:
you may consider upgrading @folio/stripes-data-transfer-components as well. It is on react v18 now

Reply:
@Terala-Priyanka, yes I think I will revert versions for jest for now. The problem inside stripes-acq-components mocks and once it will be resolved then tests packages can be updated.